### PR TITLE
Add ABE page for `<amp-bodymovin-animation>`

### PR DESCRIPTION
**Pull request by nainar**
The relevant markdown file is here: https://github.com/ampproject/amphtml/blob/master/extensions/amp-bodymovin-animation/amp-bodymovin-animation.md

### CODE REVIEW

**Comment by nainar** [comment]
@sebastianbenz could you PTAL? I added the component to the amphtml repo and am adding the relevant ABE page here.

File: src/20_Components/amp-bodymovin-animation.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<!-- 
+  ## Introduction
+
+  Embed Bodymovin animations in your AMP HTML files.
```
Comment (sebastianbenz) [inline src/20_Components/amp-bodymovin-animation.html]: Can you add a link to the official amphtml doc for the component .

File: src/20_Components/amp-bodymovin-animation.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+</head>
+<body>
+
+  <!-- ## Introduction -->
+  <!--
+    The `<amp-bodymovin-animation>` component embeds an <a href="http://airbnb.io/lottie/">AirBnB Bodymovin animation player</a>, which renders an animation from JSON generated by <a href="https://www.adobe.com/products/aftereffects.html">Adobe After Effects</a>.
```
Comment (sebastianbenz) [inline src/20_Components/amp-bodymovin-animation.html]: Can you merge this section with the Introduction section at the top.

File: src/20_Components/amp-bodymovin-animation.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  -->
+
+  <!-- ## Basic Usage -->
+  <!-- **Example 1: Embed Bodymovin animations via the `src` attribute** -->
+  <amp-bodymovin-animation
+    layout="fixed"
+    width="200" height="200"
+    src="https://nainar.github.io/loader.json">
```
Comment (sebastianbenz) [inline src/20_Components/amp-bodymovin-animation.html]: Can you also copy the json source to the src/json directory? BTW, does the json file need to be served using AMP CORS headers?

File: src/20_Components/amp-bodymovin-animation.html [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  </amp-bodymovin-animation>
+
+  <!-- ## Attributes -->
+  <!--
+
```
Comment (sebastianbenz) [inline src/20_Components/amp-bodymovin-animation.html]: nit: remove empty line

File: src/20_Components/amp-bodymovin-animation.html (hunk 2 above)
Comment (nainar) [reply src/20_Components/amp-bodymovin-animation.html]: Done!

File: src/20_Components/amp-bodymovin-animation.html (hunk 4 above)
Comment (nainar) [reply src/20_Components/amp-bodymovin-animation.html]: Done

File: src/20_Components/amp-bodymovin-animation.html (hunk 3 above)
Comment (nainar) [reply src/20_Components/amp-bodymovin-animation.html]: Done. Should I pull the JSON from the local files too? 

No it doesn't need to be served via AMP CORS headers. We do the fetching and the parsing of the JSON.

File: src/20_Components/amp-bodymovin-animation.html (hunk 1 above)
Comment (nainar) [reply src/20_Components/amp-bodymovin-animation.html]: Done. I also added a link to the supported actions.

**Comment by sebastianbenz** [comment]
The `loop` attribute doesn't validate yet, is this coming?

File: src/20_Components/amp-bodymovin-animation.html (hunk 3 above)
Comment (sebastianbenz) [reply src/20_Components/amp-bodymovin-animation.html]: Please pull the json from the local files. thanks!

Comment (nainar) [reply src/20_Components/amp-bodymovin-animation.html]: Done.

**Comment by nainar** [comment]
Yes seems like the validator changes haven't made it to prod yet? I have skipped validation and will come back and fix this up once those land?

**Pull request merged**